lib/ui_utilities/build_weather_details.dart

```diff
@@ -19,46 +20,46 @@ class BuildWeatherDetails extends StatelessWidget {
     return Padding(
       padding: const EdgeInsets.symmetric(horizontal: 8),
       child: ClipRect(
-                      child: BackdropFilter(
-                        filter: ImageFilter.blur(sigmaX: 3, sigmaY: 3),
-                        child: Container(
-                          padding: EdgeInsets.all(5),
-                          width: 110,
-                          height: 110,
-                          decoration: BoxDecoration(
-                            gradient: LinearGradient(
-                              begin: AlignmentDirectional.topStart,
-                              end: AlignmentDirectional.bottomEnd,
-                              colors: [
-                                Color(0xFF1A2344).withOpacity(0.5),
-                                Color(0xFF1A2344).withOpacity(0.2),
-                              ]),
-                              
-                          ),
-                          child: Column(
-                            mainAxisAlignment: MainAxisAlignment.center,
-                            children: [
-                              Icon(icon,
-                              color: Colors.white,
-                               ),
-                               SizedBox(height: 8,),
-                               Text(label,
-                               style: GoogleFonts.lato(
-                                fontSize: 18,
-                                color: Colors.white,
-                                fontWeight: FontWeight.bold,
-                               ), ),
-                               SizedBox(height: 8,),
-                               Text(value is String ? value: value.toString(),
-                               style: GoogleFonts.lato(
-                                fontSize: 18,
-                                color: Colors.white,
-                                
-                               ), )
-                            ],
-                          ),
-                        ),
-                        ),),
+        child: BackdropFilter(
+          filter: ImageFilter.blur(sigmaX: 3, sigmaY: 3),
+          child: Container(
+            padding: EdgeInsets.all(5),
+            width: 110,
+            height: 110,
+            decoration: forecastTileGradient,
+            child: Column(
+              mainAxisAlignment: MainAxisAlignment.center,
+              children: [
+                Icon(
+                  icon,
+                  color: Colors.white,
+                ),
+                SizedBox(
+                  height: 8,
+                ),
+                Text(
+                  label,
+                  style: GoogleFonts.lato(
```

Comment:
let's add this in styles class and use it here as a constant

Reply:
done